README.md

```diff
@@ -45,6 +45,8 @@ apiRouter.use('/cortex', await cortex(cortexEnv));
     headers:
       Authorization: Bearer ${CORTEX_TOKEN}
 ```
+> ℹ️ 
+> The ${CORTEX_BACKEND_HOST_URL} value should point to the Cortex API. If syncing to Cortex Cloud this would be `https://api.getcortexapp.com`. If running Cortex Self-Managed, please updated accordingly.
```

Comment:
```suggestion
> The ${CORTEX_BACKEND_HOST_URL} value should point to the Cortex API. If syncing to Cortex cloud this would be `https://api.getcortexapp.com`. If running Cortex self-managed, please update accordingly.
```